delivery_readmes/DELIVERY.README.RNASeq.txt

```diff
@@ -0,0 +1,119 @@
+The root folder, which is named by the NGI project ID, contains one report folder, one analysis results folder and a folder for each sample.
+Each sample folder is accompanied by a .lst-file containing a list of the files in the folder and a .md5-file containing the MD5-checksums of the files in the folder.
+Use the MD5-checksums to verify the integrity of the files after transfer.
+
+├──ProjectID
+	├── 00-Reports
+	├── 01-RNA-Results
+        ├── Sample1
+        ├── Sample1.lst
+        ├── Sample1.md5
+        ├── Sample2
+        ├── Sample2.lst
+        ├── Sample2.md5
+	...
+	...
+        ├── SampleN
+        ├── SampleN.lst
+        └── SampleN.md5
+
+
+The root folder contains following files as well:
+
+├──ProjectID
+        ├── ACKNOWLEDGEMENTS.txt
+        ├── DELIVERY.README.RNASeq.txt
+        ├── miscellaneous.lst
+        └── miscellaneous.md5
+
+--ACKNOWLEDGEMENTS.txt
+A file describing what to include in any publications using this data.
+
+--miscellaneous.lst
+A list of the files of the delivery, excluding raw data.
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+
+File integrity checks
+---------------------
+It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following:
+md5sum -c file_with_checksums.md5
+For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
+
+
+====================================================================
+ProjectID -> 00-Reports
+====================================================================
+00-Reports folder contains the reports with the sample and sequence QC. It also contains a project summary report and MultiQC reports.
+
+├── 00-Reports
+	├── ProjectName_project_summary.html
+	├── ProjectName_sample_info.txt
+	├── ProjectName_lane_info.txt
+	├── ProjectName_library_info.txt
+	├── ProjectName_multiqc_report.html
+	└── manifestFiles
+		└── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
+
+--ProjectName_project_summary.html
+A report summarizing information about the project
+
+--ProjectName_[sample/lane/library]_info.txt
+Tab-separated text files for each table found in the report, convenient for loading into Excel
+
+--ProjectName_multiqc_report.html
+Report summarizing general quality statistics based on the bioinformatics tools FastQC and FastQ Screen.
+
+--<ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
+A file summarizing the run with information about instruments and library.
+
+
+====================================================================
+ProjectID -> 01-RNA-Results
+====================================================================
+Contains best practice analysis results from the nf-core/rnaseq pipeline. Refer to
+https://nf-co.re/rnaseq/output for output description.
+
+├── 01-RNA-Results
+	├── dupradar
+	├── fastqc
+	├── featureCounts
```

Comment:
As with the pipeline version, this will change with different versions of the pipeline. We won't run featureCounts with v3.0 of the RNA-seq pipeline for example. I think it's probably better to just keep the top-level folder name.

Reply:
Removed the individual tool result directories.